source/Octopus.Server.Client/Model/Endpoints/ListeningKubernetesTentacleEndpointConfigurationResource.cs

```diff
@@ -0,0 +1,20 @@
+using Octopus.Client.Extensibility.Attributes;
+
+namespace Octopus.Client.Model.Endpoints
+{
+    public class ListeningKubernetesTentacleEndpointConfigurationResource : KubernetesTentacleEndpointConfigurationResource, IListeningTentacleEndpointResource
```

Comment:
This is a rename from `ListeningTentacleEndpointConfigurationResource` to `ListeningKubernetesTentacleEndpointConfigurationResource`. It's a mouthful but I want to make it clear that this is used for the KubernetesTentacle, not just any random Tentacle.

Reply:
I've removed the rename as it's not worth it.